diff --git a/lab2_git/HelloWorld/src/HelloWorld.java b/lab2_git/HelloWorld/src/HelloWorld.java
index b316ea1..e31d929 100644
--- a/lab2_git/HelloWorld/src/HelloWorld.java
+++ b/lab2_git/HelloWorld/src/HelloWorld.java
@@ -1,7 +1,7 @@
 
 public class HelloWorld {
 	public static void main( String [] args){
-		System.out.println("Hello World!");
+		System.out.println("Bonjour, monde");
 	}
 
 }
